core-to-plc/core-to-plc.cabal

```diff
@@ -0,0 +1,58 @@
+cabal-version: 1.18
+name: core-to-plc
+version: 0.1.0.0
+license: BSD3
+license-file: LICENSE
+copyright: Copyright: (c) 2018 Input Output
+maintainer: michael.peyton-jones@iohk.io
+author: Michael Peyton Jones
+synopsis: GHC Core to Plutus Core compiler
+description:
+    Complier that converts GHC Core to Plutus Core
+category: Language
+build-type: Simple
+extra-doc-files: README.md
+
+source-repository head
+    type: git
+    location: https://github.com/input-output-hk/plutus-prototype
+
+library
+    exposed-modules:
+        Language.Plutus.CoreToPLC.Plugin
+        Language.Plutus.CoreToPLC.Error
+    other-modules:
+        Language.Plutus.CoreToPLC
+    hs-source-dirs: src
+    default-extensions: 
+        -- should be in base
+        ScopedTypeVariables ExplicitForAll
```

Comment:
default-extensions (and probably ghc-options) should be the same across all packages in this repo. In language-plutus-core there are

```

    default-extensions: ExplicitForAll ScopedTypeVariables
                        DeriveGeneric StandaloneDeriving DeriveLift
                        GeneralizedNewtypeDeriving DeriveFunctor DeriveFoldable
                        DeriveTraversable
    other-extensions: DeriveAnyClass FlexibleContexts FlexibleInstances
                      MultiParamTypeClasses TypeFamilies OverloadedStrings
                      MonadComprehensions ConstrainedClassMethods TupleSections GADTs
                      RankNTypes TemplateHaskell QuasiQuotes TypeApplications
                      ExistentialQuantification
    ghc-options: -Wall -Wnoncanonical-monad-instances
                 -Wincomplete-uni-patterns -Wincomplete-record-updates
                 -Wredundant-constraints -Widentities
```